charts/acp-cd/templates/import.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    {{- include "acp-cd.labels" . | nindent 4 }}
+  name: {{ include "acp-cd.fullname" . }}
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+spec:
+  backoffLimit: 4
+  template:
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      volumes:
+        - name: import
+          configMap: 
+            name: {{ include "acp-cd.fullname" . }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
```

Comment:
We need args: tenant-id and server-id to be able to run acp-cd import for Authorization Server URL with vanity domain.
Otherwise it can't find proper tenant/server.

Example **Authorization Server URL**: https://acp.local:8443/system
Following error occurs:
`│ {"error":"failed to get tenant id from the path: tenant id / server is missing in the issuer url","level":"fatal","msg":"failed to init configuration client","time":"2022-02-21T15:11:16Z"}  `

Reply:
https://github.com/cloudentity/acp-helm-charts/blob/master/charts/istio-authorizer/values.yaml#L100

Reply:
@wobara-cloudentity @ppodbielski-cloudentity import cmd does not support vanity domains now.

We could potentially add support for the tenant vanity domain, but the server vanity domain for the import does not make sense.